.github/workflows/release.yaml

```diff
@@ -79,3 +79,25 @@ jobs:
           
           helm package charts/clabernetes --version $RELEASE_VERSION
           helm push clabernetes-$RELEASE_VERSION.tgz oci://ghcr.io/srl-labs/clabernetes
+
+      - name: run goreleaser to build clabverter/checksums
+        uses: goreleaser/goreleaser-action@v5
+        with:
+          version: latest
```

Comment:
quick shout out
maybe pin the version to make sure goreleaser version bumps don't break the build when we don't expect this?

Reply:
good shout, updating now!